dap-python.el

```diff
@@ -180,7 +191,14 @@ as the pyenv version then also return nil. This works around https://github.com/
   "Populate CONF with the required arguments."
   (let* ((host "localhost")
          (debug-port (dap--find-available-port))
-         (python-executable (dap-python--pyenv-executable-find dap-python-executable))
+	 (virtualenv-type (plist-get conf :virtualenv))
+         (python-executable 
+	  (cond ((= virtualenv-type "pyenv")
```

Comment:
I think `string=` is better for string comparassion

Reply:
I afraid that symbol will pollute global variables... (I know emacs-lisp a bit.)
string= is better

Reply:
If the symbol is correct in terms of emacs lisp, please tell me.

Reply:
Symbols don't pollute the global namespace, unless you intern them, which you needn't do. A symbol ('foo) is just a lisp object like an int, string, .... You could use (eq virtualenv-type 'pyenv) instead of (= virtualenv-type "pyenv"), the latter invalid anyway since = can only compare ints and markers.